pytestqt/qtbot.py

```diff
@@ -285,14 +287,14 @@ def waitSignal(self, signal=None, timeout=1000, raising=None):
                 raising = True
             else:
                 raising = _parse_ini_boolean(raising_val)
-        blocker = SignalBlocker(timeout=timeout, raising=raising)
+        blocker = SignalBlocker(timeout=timeout, raising=raising, callback=callback)
         if signal is not None:
             blocker.connect(signal)
         return blocker
 
     wait_signal = waitSignal  # pep-8 alias
 
-    def waitSignals(self, signals=None, timeout=1000, raising=None):
+    def waitSignals(self, signals=None, timeout=1000, force_order = "none", callbacks = None, raising=None):
```

Comment:
I'm OK with renaming the parameter to `order` and placing it at the end. Spaces around the default value of the parameter can be removed.